apps/docs/src/app/core/component-docs/popover/examples/popover-complex-example/popover-complex-example.component.html

```diff
@@ -17,16 +17,12 @@
             <button fd-button label="With Header"></button>
         </fd-popover-control>
         <fd-popover-body>
-            <div fd-popover-body-header>
-                <div fd-bar [cozy]="true">
-                    <div fd-bar-left>
-                        <fd-bar-element>Cozy Header</fd-bar-element>
-                    </div>
-                </div>
-            </div>
             <ul fd-list>
+				<li fd-list-group-header>
+					<span fd-list-title>Cozy Header</span>
+				</li>
                 <li fd-list-item *ngFor="let option of list">
-                    <a [attr.href]="option.url" fd-list-title>{{ option.text }}</a>
+                    <span fd-list-title>{{ option.text }}</span>
```

Comment:
I don't think this is solving the problem. You can't just change the link to a span.

Reply:
fixed